### PR TITLE
Added missing functions

### PR DESCRIPTION
**Pull request by dbthesecond**
## Type
- [ ] Bug Fix
- [x] Functions: $isChannelManageable , $isRoleEditable, $setRolePosition, $isServerDeafened, $isServerMuted, $isVideoOn, $isStreaming, $isSelfDeafened, $isSelfMuted, $readyTimestamp, $packageName
- [ ] Callbacks: \<Callback Name>
- [ ] Handlers: \<Handler Type>
- [ ] Others: \______

Dependencies (Third Party Modules) needed: <Name | Github Link> (Maximum: 20MB~ size)

Want a credit? Discord tag or other social media link: `BEAST#0001`

Referenced Issue: #NaN (Answer for an issue, if any)

## Description
Pull Request Description Here

### CODE REVIEW

**Comment by dbthesecond** [comment]
Ok done merged all the functions to 1 pr

**Comment by dbthesecond** [comment]
Ok added all required missing functions

File: package/functions/Funcs/misc/packageName.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+module.exports = d => {
+    let { code, result } = d.util.openFunc(d);
+
+    result = name;
+
+    return {
+        code: d.util.setCode({ function: d.func, code, result })
+    }
+}
```
Comment (USERSATOSHI) [inline package/functions/Funcs/misc/packageName.js:11]: this is deprecated

**Review by USERSATOSHI — CHANGES_REQUESTED** [review]

File: package/functions/Funcs/util/isChannelManageable.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = async d => {
+  const data = d.util.openFunc(d)
+  const [ id ] = data.inside.splits;
```
[the remark this answers is not in the document]
Comment (dbthesecond) [reply package/functions/Funcs/util/isChannelManageable.js]: Done

File: package/functions/Funcs/calling/setRolePosition.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
 module.exports = async d => {
     const data = d.util.openFunc( d );
     if( data.err ) return d.error( data.err );
-
-    const [ roleId,newPosition ] = data.inside.splits;
-    const role = d.guild?.roles?.cache?.get(roleID);
+    const [ roleId,newPosition,guildId = d.guild?.id ] = data.inside.splits;
+    const guild = d.util.getGuild(d,guildId);
+    if(!guild) return d.aoiError.fnError( d,'guild',{ inside : data.inside });
+    const role = guild.roles?.cache?.get(roleID);
```
[the remark this answers is not in the document]
Comment (dbthesecond) [reply package/functions/Funcs/calling/setRolePosition.js:7]: Ooo ok

File: package/functions/Funcs/util/isChannelManageable.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  const channel = id ? d.guild?.channels?.cache?.get(id)?.id : d.channel?.id;
```
[the remark this answers is not in the document]
Comment (dbthesecond) [reply package/functions/Funcs/util/isChannelManageable.js]: Done

**Comment by dbthesecond** [comment]
Done all the changes u said. Now see if i need anything more to do

File: package/functions/Funcs/misc/packageName.js (hunk 1 above)
Comment (dbthesecond) [reply package/functions/Funcs/misc/packageName.js:11]: So do i remove it?

Comment (dbthesecond) [reply package/functions/Funcs/misc/packageName.js:11]: I just added according to the changelog of v4.5.0

Comment (USERSATOSHI) [reply package/functions/Funcs/misc/packageName.js:11]: > So do i remove it?

yes remove it @dbthesecond u can see its -$packageName in changelog

**Review by USERSATOSHI — APPROVED** [review]

**Comment by dbthesecond** [comment]
Ok

**Comment by dbthesecond** [comment]
Ok done removed $packageName

**Comment by dbthesecond** [comment]
Now u can merge

**Pull request merged**